### PR TITLE
Showing profile picture in base template + Index page shows description of activity only 2 lines

### PR DESCRIPTION
**Pull request by Thanawas-Sirilertsathit**
- Dropdown feature in base template header using profile picture and name.
- Index page shows description of activity only 2 lines after that it will be ...
- Profile picture will only be shown when login.

### CODE REVIEW

**Review by OmegaOoh — CHANGES_REQUESTED** [review]
I tried on newly created database and got this error
Notfound Error

The color of dropdown is hard to read.
use `base-300` as dropdown color and `neutral` as the color while hover. 

also change the view to use REST Framework.

**Comment by Thanawas-Sirilertsathit** [comment]
> I tried on newly created database and got this error Notfound Error
> 
> The color of dropdown is hard to read. use `base-300` as dropdown color and `neutral` as the color while hover.
> 
> also change the view to use REST Framework.

I don't see any Runtime error. I tried creating new database already, it can show profile picture correctly.

**Comment by Thanawas-Sirilertsathit** [comment]
I got only 403, 400 when forgot to setup SITE correctly

File: mysite/views.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-def is_authen(request: HttpRequest) -> JsonResponse:
-    """Return Json which inform that user is login right now."""
-    print(request.user.is_authenticated)
-    return JsonResponse({"is_authen": request.user.is_authenticated})
+@decorators.api_view(["GET"])
+def profile_picture_view(request: HttpRequest) -> response.Response:  # pragma: no cover
+    """Return profile picture url from Google account."""
+    user = request.user
+    try:
+        social_account = SocialAccount.objects.get(user=user, provider=OAUTH_PROVIDER)
```
Comment (OmegaOoh) [inline mysite/views.py:13]: You can remove provider, we got only 1. If you consider remove this you may need to remove OAUTH_PROVIDER from `settings.py` and `sample.env` also

**Review by OmegaOoh — CHANGES_REQUESTED** [review]

File: mysite/views.py (hunk 1 above)
Comment (OmegaOoh) [reply mysite/views.py]: @opxz7148 [fixed it.](https://github.com/OmegaOoh/ku-tangtee/pull/64/commits/521eafdb83c009e0b5591ab5e9a4bec6137b6cd4)

**Review by OmegaOoh — CHANGES_REQUESTED** [review]
sample.env and settings.py still contiains unused OAUTH_PROVIDER.

**Review by opxz7148 — APPROVED** [review]
Look good

**Review by TeerapatTrepopsakulsin — COMMENTED** [review]
- Dropdown feature in base template header using profile picture and name. _Approve_
- Index page shows description of activity only 2 lines after that it will be ...
test case:
```
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```
got:
image

- Profile picture will only be shown when login. _Approve_

**Comment by Thanawas-Sirilertsathit** [comment]
I mean in Activity Index page detail will be shorten to 2 lines

**Comment by Thanawas-Sirilertsathit** [comment]
I will fix the Activity Detail page when I have to do profile pic of participants.

**Review by OmegaOoh — APPROVED** [review]

**Review by opxz7148 — APPROVED** [review]
Somehow its profile pic doesn't appear on Firefox. But it's work for chrome but I think it's good to merge.

**Pull request merged**